importccl: fix sqlbase refactor rebase issue with IMPORT PGDUMP

otan opened the pull request:
Looks like 8e5e54f9270d7281184282d5946bd6e2ce6e9485 incorrectly modified
the `idx` back to `i` causing a bug. This was not caught
as `TestImportData` is skipped (I will fix this separately) -- I
tested this with it unskipped (by hand) beforehand.

Release justification: bug fixes and low-risk updates to new functionality

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/53807)
<!-- Reviewable:end -->

otan committed:
importccl: fix sqlbase refactor rebase issue with IMPORT PGDUMP

Looks like 8e5e54f9270d7281184282d5946bd6e2ce6e9485 incorrectly modified
the `idx` back to `i` causing a bug. This was not caught
as TestImportData is skipped (I will fix this separately) -- I
tested this with it unskipped (by hand) beforehand.

Release justification: bug fixes and low-risk updates to new functionality

Release note: None

otan commented:
unskipped some tests in https://github.com/cockroachdb/cockroach/pull/53808

ajwerner reviewed (approved):
good catch, thanks!

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/53807#-:-MGBuIhV7vyi6mQrgzeY:bjzmcav)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @miretskiy)



<!-- Sent from Reviewable.io -->

otan commented:
bors r=ajwerner
